standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter_FractionalSuite.scala

```diff
@@ -207,4 +214,169 @@ class StandardizationInterpreter_FractionalSuite extends FunSuite with SparkTest
     assertResult(exp)(std.as[FractionalRow].collect().sortBy(_.description).toList)
   }
 
+  test("No pattern, but altered symbols") {
+    val input = Seq(
+      ("01-Positive", "+3"),
+      ("02-Negative", "~8123,4"),
+      ("03-Null", null),
+      ("04-Big", "7899012345678901234567890123456789012346789,123456789"),
+      ("05-Big II", "+1E40"),
+      ("06-Big III", "2E308"),
+      ("07-Small", "~7899012345678901234567890123456789012346789,123456789"),
+      ("08-Small II", "~1,1E40"),
+      ("09-Small III", "~3E308"),
+      ("10-Wrong", "hello"),
+      ("11-Infinity", "+∞"),
```

Comment:
No, we don't. Using Pattern doesn't support the keyword. So it would create strange inconsistency.
Yes, the string for redifinition of the _infinity_ could be "opened" in `Metadata`. All properties from `DecimalSymbols` could be. After some feedback, I made only the most basic ones (IMO) changeable. We can discuss, if others would be useful too.

Reply:
Let's discuss on CQC

Reply:
Where did we end with this?

Reply:
We didn't get to it. Wonder about the best format to discuss theses. 🤔

Reply:
But, it actually can be an extra PR. Don't have to be part of this.

Reply:
Ok, next issue #1034